snuba/query/timeseries_extension.py

```diff
@@ -90,12 +90,20 @@ def process_query(
                 BooleanFunctions.AND,
                 binary_condition(
                     ConditionFunctions.GTE,
-                    Column(None, None, self.__timestamp_column),
+                    Column(
+                        f"_snuba_{self.__timestamp_column}",
+                        None,
+                        self.__timestamp_column,
+                    ),
```

Comment:
Same concern. are these changes related with comparing the SQL. 
Also why do we need to add an alias to this column ?

Reply:
SnQL mangles the alias for every expression. The regular query parser does too, but these extension conditions are added after the mangling is done. It was easier to add the alias here than to try and skip them in SnQL.